backend/src/main/java/service/UserService.java

```diff
@@ -16,7 +17,7 @@ public void addUser(String addUserQuery) {
             Statement statement = databaseConnection.connection();
             statement.execute(addUserQuery);
             statement.close();
-            System.out.println("service.User has been correctly added.");
+            System.out.println("Model.User has been correctly added.");
```

Comment:
"User has been correctly added" -> there is no need to write the package name

Reply:
Automatic refactoring made it! Removed.